### PR TITLE
Correct diagram file link

### PR DESCRIPTION
**Pull request by lecourtoisflo**

### CODE REVIEW

File: sources/Outputs/include/Par.h [hunk 1]
```diff
@@ -42,14 +42,17 @@ class Par {
      *
      * @param base basename for current simulation
      * @param filename file path for output PAR file (corresponds to basename)
+     * @param dirname the dirname of the output PAR file
```
Comment (rosiereflo) [inline sources/Outputs/include/Par.h:45]: Wrong parameter name and wrong parameter order (name 'dir' below)

**Review by rosiereflo — CHANGES_REQUESTED** [review]

File: sources/Context.cpp [hunk 2]
```diff
@@ -151,15 +151,15 @@ Context::exportOutputs() {
   // create specific par
   file::path parOutput(config_.outputDir());
   parOutput.append(basename_ + ".par");
-  outputs::Par parWriter(outputs::Par::ParDefinition(basename_, parOutput.generic_string(), generators_));
+  outputs::Par parWriter(outputs::Par::ParDefinition(basename_, config_.outputDir(), parOutput.generic_string(), generators_));
```
Comment (marcochiaramello) [inline sources/Context.cpp:154]: Here I have the impression we pass twice an information, config_.outputDir() and  parOutput.generic_string().
Indeed, parOutput.genereic_string() is built just above from config_.outputDir()

**Review by marcochiaramello — CHANGES_REQUESTED** [review]

File: sources/Context.cpp (hunk 2 above)
Comment (lecourtoisflo) [reply sources/Context.cpp:154]: technicay correct, as parOutput = config_.outputDir() + basename_. I keep all of these to keep the spirit of putting the filename in the writer definition and to have flexibility in the future if, for one reason or another we would put the diagram file in another directory than the dyd and par. Should I restrict the information ?

File: sources/Outputs/include/Par.h (hunk 1 above)
Comment (lecourtoisflo) [reply sources/Outputs/include/Par.h:45]: OK

**Review by rosiereflo — APPROVED** [review]

File: sources/Context.cpp (hunk 2 above)
Comment (rosiereflo) [reply sources/Context.cpp:154]: I don't have any preference, both point of views are OK to me.... (sorry for not helping ;))
Marco I'll let you have the final word on this.

Comment (marcochiaramello) [reply sources/Context.cpp:154]: ok, let's keep some room for flexibility :)

File: tests/outputs/TestPar.cpp [hunk 3]
```diff
@@ -58,7 +59,7 @@ TEST(TestPar, write) {
                                                                      },
                                                                      4., 40., 44., 440.)};
 
-  dfl::outputs::Par parWriter(dfl::outputs::Par::ParDefinition(basename, outputDir + "/" + filename, generators));
+  dfl::outputs::Par parWriter(dfl::outputs::Par::ParDefinition(basename, dirname, outputDir + "/" + filename, generators));
```
Comment (marcochiaramello) [inline tests/outputs/TestPar.cpp:62]: small question: explicit separator like here "/" will work on OS platforms?

**Review by marcochiaramello — CHANGES_REQUESTED** [review]

File: tests/outputs/TestPar.cpp (hunk 3 above)
Comment (lecourtoisflo) [reply tests/outputs/TestPar.cpp:62]: I don't know but it is true that we can protect ourselves in that case. I'll do the correction. But we'll still have the issue in the config.json if we have multi-directories output dir

**Review by marcochiaramello — APPROVED** [review]

**Review by rosiereflo — APPROVED** [review]

**Review by PerninGuillaume — APPROVED** [review]

**Review by gautierbureau — APPROVED** [review]

**Pull request merged**